### ISSUE TITLE
Min/Max/Close Buttons not visible

### ISSUE THREAD

**Issue by DDGo** [muckSponge/MaterialFox#108]
![image](https://user-images.githubusercontent.com/5735558/58803555-174c8400-8610-11e9-8125-41dc67274edb.png)

Somehow after I install MaterialFox my Min/Max/Close buttons are invisible, until you hover over them. I have reset my profile to see if works, and it doesn't. Only after disabling the global.css in userchrome.css I get the normal Windows 10 buttons back.

**Comment by muckSponge** [comment]
What version of Firefox are you using?

**Comment by Visnes** [comment]
Same problem here. Running Firefox 67.0 64-bit on Windows 10.

**Comment by DDGo** [comment]
@muckSponge I have tried 3 different versions of Firefox on Windows:

- Firefox Stable (67) - doesn't work
- Firefox Beta (68) - works
- Firefox Nightly (69) doesn't work

All of these installations are using a fresh different profile.

**Comment by Schanzation** [comment]
Same issue for me. Using FF 67 64-bit on Windows 10 Pro.
+ another issue I marked in red in this picture:
https://imgur.com/a/kBm3rJp

On a side note, can you make the height between the bookmark drop-down menu and extension drop-down menu closer together?

**Comment by muckSponge** [comment]
Oops, don't even know how I managed to miss that - last few commits were directly from a Windows 10 machine running 67! I've removed the offending code, which may cause a minor regression elsewhere in the UI.

**Comment by DDGo** [comment]
@muckSponge Looks like it works again for me 👍 Haven't seen the minor regression you are talking about (maybe it's not that obvious to me)

**Comment by muckSponge** [comment]
I know I put that block of code in for a reason but can't recall why. Something popup-related. If you find anything subtle (an icon not coloured like the rest) open up an issue for it, otherwise we're all good.
